.buildkite/pipeline.yml

```diff
@@ -1,6 +1,7 @@
 agents:
-  queue: central
-  modules: julia/1.10.0 cuda/12.2 ucx/1.14.1_cuda-12.2 openmpi/4.1.5_cuda-12.2 nsight-systems/2023.3.1
+  queue: new-central
+  slurm_mem: 8G
+  modules: climacommon/2024_02_20
```

Comment:
You can use climacommon/2024_02_27 now and remove `OMPI_MCA_opal_warn_on_missing_libcuda: 0` below

Reply:
Also, please remove the JuliaProject.toml file